generators/server/templates/src/main/resources/config/liquibase/changelog/initial_schema.xml.ejs

```diff
@@ -163,15 +163,24 @@
                   <%_ if (devDatabaseType === 'mssql' || prodDatabaseType === 'mssql') { _%>
                   identityInsertEnabled="true"
                   <%_ } _%>
-                  tableName="<%= jhiTablePrefix %>_authority"/>
+                  tableName="<%= jhiTablePrefix %>_authority">
+            <%_ if (devDatabaseType === 'oracle' || prodDatabaseType == 'oracle') { _%>
```

Comment:
I think all these conditions (including existing ones) `devDatabaseType === X || prodDatabaseType === X` can be replaced by just `prodDatabaseType === X`.

BTW: here an extra `=` is missing in `prodDatabaseType == 'oracle'`

Reply:
Thanks for the review @murdos! both dev and prod db types need to be checked otherwise it won't work if the user chooses oracle for dev.

Reply:
The only way to choose Oracle for dev is to choose Oracle for prod as well.  [The prompt](https://github.com/jhipster/generator-jhipster/blob/2e448adb31bbc063d1f5ee98e110892c73695e8b/generators/server/prompts.js#L206-L217) only allows H2 and the prodDatabseType